pkg/authn/google/auth.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2018 Google LLC All Rights Reserved.
```

Comment:
is this intentionally 2018 vs. 2022?  It doesn't show up as a move.

Reply:
This is a move, whether Git sees it as one or not. I'd rather not change anything.